ForkClone.md

```diff
@@ -1,33 +1,33 @@
-# Fork and Clone 
+# Fork and clone 
 
-## Steps to fork the repo: 
+## Steps to fork the repository (repo): 
 
 1. Go to the main GitHub page of the repo:
     (https://github.com/gbowne1/codebooker/)
    
-1. Once you are on the main GitHub page of the repo, click the **Fork** button at the top right of the page. 
+1. Select **Fork** at the top right of the page. 
 
     ![](fork.png)
 
-1. You will now be on a page with the heading **Create a new fork**.
+    This opens a page with the heading **Create a new fork**.
```

Comment:
I made this part of an existing step instead of a new step because it doesn't require the user to do anything. I also changed the future tense to present tense because that's most common in tech writing.